### ISSUE TITLE
Hugo extended edition required by default?

### ISSUE THREAD

**Issue by mloskot** [theNewDynamic/gohugo-theme-ananke#678]
I've read and learned this
https://github.com/theNewDynamic/gohugo-theme-ananke/blob/91df000ca82769a7578df81fa142d1a18e446bc0/README.md?plain=1#L275-L277

but using the latest 
```bash
$ hugo version
hugo v0.123.8-5fed9c591b694f314e5939548e11cc3dcb79a79c linux/amd64 BuildDate=2024-03-07T13:14:42Z VendorInfo=gohugoio
```

and following the Hugo getting started tutorial together with 

```bash
git submodule add https://github.com/budparr/gohugo-theme-ananke.git themes/ananke
echo 'theme = "ananke"' >> hugo.toml
```

I'm experiencing this:

![image](https://github.com/theNewDynamic/gohugo-theme-ananke/assets/80741/8d6c3993-2f28-45f4-85f8-cabc1ff57ab1)

Is this expected?

Replacing the standard Hugo with the extended Hugo fixes the problem, of course.

**Comment by davidsneighbour** [comment]
I always use the extended version for everything and think of the "regular" version as one we would use on slower, smaller systems where we have a finished theme and only content to transform.

I think the best solution in the future (in v2.12) will be to have a complete theme set compiled in static and, if we are working with the extended version of Hugo, to use the pipelines. 

I am not 100% sure if that will work out of the box without configuration.

**Comment by aetos382** [comment]
The key feature of the extended version is its support for LibSass; however, the built-in LibSass has already been deprecated, and the use of dart-sass is recommended.
https://gohugo.io/installation/linux/

All themes must be compatible with the standard version and dart-sass.

**Comment by davidsneighbour** [comment]
> All themes must be compatible with the standard version and dart-sass.

That is false as per [Theme configuration section](https://github.com/gohugoio/hugoThemesSiteBuilder#theme-configuration) on [hugoThemesSiteBuilder](https://github.com/gohugoio/hugoThemesSiteBuilder).

Nonetheless as can be seen from the connected PRs and issues this is worked on.

**Comment by davidsneighbour** [comment]
Resolved in [v2.16.0](https://github.com/gohugo-ananke/ananke/releases).

As of this version Extended Hugo is not a requirement anymore.
